src/Nancy.Testing.Tests/ConfigurableBootstrapperFixture.cs

```diff
@@ -219,5 +219,21 @@ public BlowUpModule()
                 Get["/"] = _ => { throw new InvalidOperationException("Oh noes!"); };
             }
         }
+
+        private static class Record

```

Comment:
There's also the `Record` class which does exactly this; https://github.com/xunit/xunit/blob/dd20821043bcb23eeecd03d0468f08c4b5378c0b/src/xunit.core/Record.cs